### PR TITLE
Fix: gesund-aktiv

### PR DESCRIPTION
**Pull request by jknndy**
& new test data

### CODE REVIEW

File: recipe_scrapers/gesundaktiv.py [hunk 1]
```diff
@@ -8,32 +7,15 @@ class GesundAktiv(AbstractScraper):
...
+            "div", {"class": "news-recipes-indgredients"}
         )
+        ingredients = ingredients_container.findAll("li")
+        return [normalize_string(ingredient.get_text()) for ingredient in ingredients]
+
+    def author(self):
+        author_tag = self.soup.find("meta", {"property": "og:site_name"})
+        if author_tag:
+            return normalize_string(author_tag.get("content"))
+        return None
```
Comment (jayaddison) [inline recipe_scrapers/gesundaktiv.py:21]:
```suggestion
    def author(self):
        return self.site_name()
```

File: recipe_scrapers/gesundaktiv.py [hunk 2]
```diff
@@ -8,32 +7,12 @@ class GesundAktiv(AbstractScraper):
...
-        return "\n".join(
-            [normalize_string(instruction.get_text()) for instruction in instructions]
-        )
-
-    def description(self):
-        return normalize_string(
-            self.soup.find("meta", {"name": "description"})["content"]
-        )
+    def author(self):
+        return self.site_name()
```
Comment (jayaddison) [inline recipe_scrapers/gesundaktiv.py:18]: Let's also move this method to the top of the class definition?

(I'm not suggesting we enforce the same ordering as JSON... yet :))

Comment (jknndy) [reply recipe_scrapers/gesundaktiv.py:18]: This may be a nice cleanup & I'd think pretty easy to implement!

**Review by jayaddison — APPROVED** [review]
Looks good to me :+1:

**Pull request merged**